src/esp/core/Utility.h

```diff
@@ -17,34 +19,65 @@
 
 namespace esp {
 namespace core {
+namespace Mn = Magnum;
```

Comment:
Try to not do this in a header, it causes the generated docs to not link to Magnum types correctly.